manifests/params.pp

```diff
@@ -10,7 +10,7 @@
   $package_ensure    = 'latest'
   $ui_package_name   = 'consul_ui'
   $ui_package_ensure = 'latest'
-  $version = '0.5.0'
+  $version = hiera('consul::version', '0.5.0')  # ensure defaults resolve from hiera as well
```

Comment:
Are we still blocked on this? Can we keep this the way it was (without hiera) and merge it?